Core.ApplicationServices/Authorization/UnauthenticatedAuthorizationContext.cs

```diff
@@ -49,5 +50,10 @@ public bool HasPermission(Permission permission)
         {
             return false;
         }
+
+        public UserDeletionStrategyType GetUserDeletionStrategy(User user)
+        {
+            return UserDeletionStrategyType.Local;
```

Comment:
again this should tell you something is wrong.. for unauthenticated case there is no permissions